packaging/linux/deb/debroot.go

```diff
@@ -136,20 +131,12 @@ func Debroot(ctx context.Context, sOpt dalec.SourceOpts, spec *dalec.Spec, worke
 	installers := createInstallScripts(worker, spec, dir, target)
 
 	const (
-		sourceFormat  = "3.0 (quilt)"
-		sourceOptions = "create-empty-orig"
+		sourceFormat = "3.0 (quilt)"
 	)
 
 	debian := base.
 		File(llb.Mkdir(filepath.Join(dir, "source"), 0o755), opts...).
-		With(func(in llb.State) llb.State {
-			if len(spec.Sources) == 0 {
-				return in
-			}
-			return in.
-				File(llb.Mkfile(filepath.Join(dir, "source/format"), 0o640, []byte(sourceFormat)), opts...).
-				File(llb.Mkfile(filepath.Join(dir, "source/options"), 0o640, []byte(sourceOptions)), opts...)
-		}).
+		File(llb.Mkfile(filepath.Join(dir, "source/format"), 0o640, []byte(sourceFormat)), opts...).
```

Comment:
Can we make sure there is an integration test case where there are no sources?

Reply:
There is one test at least that runs without sources. I encounter this as previously PR was failing on not having the upstream tar for dpkg, as we were skipping the taring if not sources where there. So I assumed that case was cover.
ref to test wtihout sources: https://github.com/Azure/dalec/blob/main/test/linux_target_test.go#L150